_dashboards/reporting-cli.md

```diff
@@ -0,0 +1,248 @@
+---
+layout: default
+title: Create reports with the Reporting CLI
+nav_order: 25
+---
+
+# Create reports with the Reporting CLI
+
+You can create Dashboard reports programmatically with the Reporting CLI without using the OpenSearch Dashboards and the Reporting plugin. This provides automation for reporting that can be connected to email workflows.
+
+For any Dashboard view you can request a report in CSV, PNG, or PDF format that is sent to an email address. This can be useful to send to multiple email recipients with an email alias.
+
+With the Reporting CLI, you can specify options for your report in the command line. The report is sent to an email address as a PDF attachment by default. You can also request a PNG image or a CSV file with the `--formats` argument, or embed the image in the email body instead of attaching it.
+
+You can download the report to the directory that you run the Reporting CLI, or you can get the report emailed by specifying SES or SMTP for the email transport option.
+
+If you want to download a CSV file, you need to have the Reporting plugin installed.
+{: .note }
+
+You can connect to OpenSearch with any of the following authentication types:
+
+- **Basic** – Basic HTTP authentication. Use `-a basic`.
+- **Cognito** – AWS Identity access with Amazon Cognito. Use `-a cognito`.
+- **SAML** – Provides authentication between an identity provider and a service provider. Use `-a saml`. OKTA provides the SAML Third-party authentication.
+- **No auth** – No authentication bypass authentication. Use `-a none`. It also defaults to No auth if `-a` flag is not specified.
+
+To learn more about Amazon Cognito, see [What is Amazon Cognito?](https://docs.aws.amazon.com/cognito/latest/developerguide/what-is-amazon-cognito.html).
+### Bypass authentication option
+
+The reporting CLI tool allows you to integrate it into your own workflow or environment so that you can bypass authentication or potential security issues. For example, if you use the Reporting CLI tool within an AWS Lambda instance, no security issues would occur as long as you run the Reporting plugin in OpenSearch Dashboards. In this case, you would use "No auth" to bypass the authentication process. To specify "No Auth" use `--auth none` in your request. Lambda users should test to make sure they can bypass access to Dashboards without credentials using No Auth.  
```

Comment:
is there any reference for bypass authentication? Need to test this.

Reply:
this information was provided in the feature brief. who can verify this was tested?

Reply:
commenting out this section until we get confirmation it works. thanks

Reply:
I'll test this. We can remove comment after testing. Thanks!